CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@
 * Commit, do not mess with rakefile, version, or history.
   (if you want to have your own version, that is fine but bump version in a commit by itself I can ignore when I pull)
 * Send a pull request. Bonus points for topic branches.
+  * Your branch must pass tests: run `rake` or `rspec spec/{file or functionality you modified}_spec.rb`.
```

Comment:
I'd just recommend the `rake` as `rspec spec/` will fail the 100% coverage requirement.

Reply:
Also how about: `Your PR must pass all existing tests with 100% code coverage: run rake`